### ISSUE TITLE
Support variables in vscode config

### ISSUE THREAD

**Issue by ex-nerd** [hbenl/vscode-mocha-test-adapter#154]
Neither `mochaExplorer.nodePath` nor `mochaExplorer.mochaPath` seem to support variable substitution in their values.  Since `mochaPath` won't look for mocha on the `$PATH` variable, I was hoping to use something like `"${env:MY_MOCHA_PATH}"`  (or even `"${workspaceFolder:MY_ROOT_PROJECT}/custom-node-path/mocha"`) but no form of variable substitution works, and mocha-test-adapter attempts to do things like `spawn ${env:MY_NODE_PATH}` instead of actually interpreting the value within that variable.

Alternately, regular old hard-coded environment variables (e.g. `$MOCHAEXPLORER_MOCHAPATH`) could be used to bypass both the config and vscode settings.

Basically, I have a large multi-root project with node installed in the main project (via `n`) along with `mocha`, `ts-node` and other shared dependencies.  Each individual sub-project is then configured separately for mocha test discovery because each set of tests needs to be run from the sub-project's directory so that the sub-project's `node_modules/` directory is accessible and then I don't need to have a dozen or more copies of `mocha` and other common utilities installed (or worry about where different team members might have installed global copies).

**Comment by allanpaiste** [comment]
This is somewhat small, yet annoying issue indeed when it comes to sharing pre-baked configuration with others.
